wrappers/python/pyrsutil.cpp

```diff
@@ -51,7 +51,7 @@ void init_util(py::module &m) {
     };
 
     m.def("rs2_project_color_pixel_to_depth_pixel", cp_to_dp, "Given pixel coordinates of the color image and a minimum and maximum depth, compute the corresponding pixel coordinates in the depth image. Returns [-1 -1] on failure.",
-          "data"_a, "depth_scale"_a, "depth_min"_a, "depth_max"_a, "depth_intrin"_a, "color_intrin"_a, "depth_to_color"_a,
-          "color_to_depth"_a, "from_pixel"_a);
+          "data"_a, "depth_scale"_a, "depth_min"_a, "depth_max"_a, "depth_intrin"_a, "color_intrin"_a, "color_to_depth"_a,
```

Comment:
@rhaschke, these changes have already been merged to development branch. We can remove these for this PR.

Reply:
@Arun-Prasad-V, these changes sneaked in when @maloel changed the base branch from master to development. As I cannot change the source branch, I opened a new PR: #12361. Closing here.

You should document the desired base branch for PRs, e.g. using [PR templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository), to avoid such confusion :wink: